src/OpenColorIO/GpuShaderClassWrapper.h

```diff
@@ -52,6 +52,7 @@ class MetalShaderClassWrapper : public GpuShaderClassWrapper
     std::string getClassWrapperFooter(const std::string& originalFooter) final;
     
     std::unique_ptr<GpuShaderClassWrapper> clone() const final;
+    const MetalShaderClassWrapper& operator=(const MetalShaderClassWrapper& rhs);
```

Comment:
The return instance is not const for assignation operator, the method signature is then
`MetalShaderClassWrapper& operator=(const MetalShaderClassWrapper& rhs);`

Reply:
This is the last issue I think @Morteeza

Reply:
@hodoulp change applied.